web/templates/settings/admin_invites_list.hbs

```diff
@@ -37,7 +37,6 @@
           intent="danger"
           custom_classes="revoke tippy-zulip-delayed-tooltip"
           data-tippy-content=(t "Revoke")
-          disabled=disable_buttons
```

Comment:
Revoke button was disabled for non-owners when invite was configured to have the new user join as "Owner". Removing this will break that.

Reply:
This condition is used for disabling revoke button -
```js
item.disable_buttons =
                item.invited_as === settings_config.user_role_values.owner.code &&
                !current_user.is_owner;
```

And then there is a different condition for edit button, but a single variable is used.